integration/spark2/src/main/spark2.1/CarbonSessionState.scala

```diff
@@ -172,6 +173,10 @@ class CarbonSessionState(sparkSession: SparkSession) extends HiveSessionState(sp
       conf,
       newHadoopConf())
   }
+
+  def getCarbonEnv() : CarbonEnv = {
+    catalog.carbonEnv
```

Comment:
Why make it session based instead of a global CarbonEnv?

Reply:
Removed. Not Needed. Temporarily added.